README.md

```diff
@@ -104,6 +104,8 @@ forever-service version 0.x.x
                                        
     --logrotateMax [value]             Maximum logrotated files to retain, default 10 (logrotate parameter)
                                        
+    --dateExt                          Archive old versions of log files adding a daily extension like YYYYMMDD instead of simply adding a number
```

Comment:
@perusworld can you rename this to logrotateDateExt, i think it is better to keep all logrotate options with logrotate prefix to be consistent